### PR TITLE
[doc] Document the Statistics class

### PR DESCRIPTION
**Pull request by yuanming-hu**
Related PR = #788 

[[Click here for the format server]](http://kun.csail.mit.edu:31415/)

### CODE REVIEW

File: docs/internal.rst [hunk 1]
```diff
@@ -1,14 +1,39 @@
...
+In some cases, it is helpful gather certain quantitative information about internal events during
+Taichi program execution. The ``Statistics`` class is designed for this purpose.
+
+Usage:
+
+::
+
+    #include "taichi/util/statistics.h"
+
+    taichi::stat.add("codegen_offloaded_tasks"); // add 1.0 to
```
Comment (xumingkuan) [inline docs/internal.rst:29]: What is "add 1.0 to"?

Comment (yuanming-hu) [reply docs/internal.rst]: Sorry I was clearly in a rush. Fixed now. Thanks :-)

**Comment by xumingkuan** [comment]
Does `print_statistics` work now? I got `AttributeError: module 'taichi_core' has no attribute 'print_statistics'`.

**Comment by yuanming-hu** [comment]
Sorry - could you try `ti.core.print_stat()`?

**Comment by xumingkuan** [comment]
`ti.core.print_stat()` works for me.

**Pull request merged**